pkg/webhook/v1beta1/experiment/validator/validator.go

```diff
@@ -67,6 +67,16 @@ func (g *DefaultValidator) InjectClient(c client.Client) {
 // ValidateExperiment validates experiment for the given instance.
 // oldInst is specified when experiment is edited.
 func (g *DefaultValidator) ValidateExperiment(instance, oldInst *experimentsv1beta1.Experiment) error {
+	namingConvention, _ := regexp.Compile("^[a-z]([-a-z0-9]*[a-z0-9])$")
+	if !namingConvention.MatchString(instance.Name) {
+		msg := fmt.Sprintf("Name must consist of lower case alphanumeric characters or '-'," +
```

Comment:
define as a constant?

Reply:
you mean the whole error message should be defined as a const at [here](https://github.com/kubeflow/katib/blob/48faabc8a48f9d4b4431e74c1d46ac5496d5382a/pkg/webhook/v1beta1/common/const.go#L19) ?

or such regex pattern should be defined as a const ?

Reply:
I think since you are not using substitution here, you can just define msg as a string.
```
msg := "Name must consist of lower case alphanumeric characters or '-'," +
" start with an alphabetic character, and end with an alphanumeric character" +	
" (e.g. 'my-name', or 'abc-123', regex used for validation is '[a-z]([-a-z0-9]*[a-z0-9])?')")
```

Reply:
I got it. Thanks !